src/service/viewport-impl.js

```diff
@@ -510,6 +502,32 @@ export class Viewport {
   }
 
   /**
+   * @param {Event} event
```

Comment:
It's not an Event... Copy the type definition that we used for `viewportObservable` in the `viewer-impl.js`.